src/components/login.tsx

```diff
@@ -0,0 +1,35 @@
+import { FormEvent, useState } from "react";
+
+function Login() {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleSubmit = (e: FormEvent) => {
```

Comment:
**suggestion**: Consider adding error handling for the form submission to improve user experience.